### PR TITLE
Update Explainer for 2024

### PR DESCRIPTION
**Pull request by iadawn**
This is a direct copy and bit of a clean from the baseline in https://github.com/w3c/silver/tree/main/explainer

### CODE REVIEW

File: .DS_Store
Comment (fstrr) [inline .DS_Store:1]: This file should be deleted—it's a macOS metadata thing. I've added a PR to update the .gitignore file to ignore these in the future.

File: explainer/index.html [hunk 1]
```diff
@@ -11,46 +11,66 @@
 
 <body>
     <section id='abstract'>
-        <p>This Explainer accompanies the drafts of the W3C Accessibility Guidelines (WCAG) 3.0.<p>
+        <p>This Explainer accompanies the drafts of the W3C Accessibility Guidelines (WCAG) 3.0.
+        <p>
```
Comment (fstrr) [inline explainer/index.html:15]:
```suggestion
        </p>
```

File: explainer/index.html [hunk 2]
```diff
@@ -11,46 +11,66 @@
...
         <h2>Goals</h2>
-        <p>The goal of WCAG is to provide information that can be used to improve the accessibility of products on a variety of platforms. WCAG 3.0 will use a model that allows it to:</p>
+        <p>The goal of WCAG is to provide information that can be used to improve the accessibility of products on a
+            variety of platforms. WCAG 3.0 will use a model that allows it to:</p>
         <ul>
             <li>address more disability needs than WCAG 2,</li>
-            <li>incorporate publishing requirements and emerging technologies such as web XR (augmented, virtual and mixed reality) and voice input,</li>
-            <li>facilitate maintenance, so that the new model will be more enduring over time as technologies evolve, and</li>
-            <li>include additional information about the ways web technologies need to work with authoring tools, user agents, and assistive technologies.</li>
+            <li>incorporate publishing requirements and emerging technologies such as web XR (augmented, virtual and
```
Comment (fstrr) [inline explainer/index.html:46]:
```suggestion
            <li>incorporate publishing requirements and emerging technologies such as web XR (augmented, virtual, and
```

**Comment by rachaelbradley** [comment]
### Review requested by 5 November 2024

Please review the [WCAG 3 Explainer (preview)](https://deploy-preview-116--wcag3.netlify.app/explainer/) and add suggested changes and comments to this Pull Request or [google document ](https://docs.google.com/document/d/1_vWUteDqNAXDYrcwylOoPt0U2hOrbjo1DzsTEs-ScFs/edit?usp=sharing)before 5 November 2024.

**Comment by alastc** [comment]
In [section 5.2](https://deploy-preview-116--wcag3.netlify.app/explainer/#requirements-and-methods), could we add a note such as: "The group is considering the addition of 'Recommendations' with the supplemental requirements, things which are good to do but may not be objectively testable."

File: explainer/index.html [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+            <li>Submitting tax information; and</li>
+            <li>Chatting with other users in a virtual reality environment.</li>
+        </ul>
+        <p>The product is the combination of all items, views, and task flows that comprise the web site, set of web
+            pages, web app, etc.</p>
+
+        <h4>Conditions</h4>
+        <p>Some tests only apply in certain situations. Testing may occasionally require determining and referencing
+            which specifications are being tested. Methods will note whether a test always applies or under what
+            conditions a test applies. Both quantitative and qualitative tests can be conditional.</p>
```
Comment (mattgarrish) [inline explainer/index.html:379]:
```suggestion
            conditions a test applies. Both quantifiable and qualitative tests can be conditional.</p>
```

File: explainer/index.html [hunk 4]
```diff
@@ -0,0 +1,553 @@
...
+    <h4>Testing Assertions</h4>
+    <p>The quality of an assertion can be tested based on how well the assertion meets the documentation
+        requirements for assertions (See <a>Documenting Assertions</a>). Conforming to WCAG does not require testing
+        supporting documentation; however, organizations may decide to adopt additional documentation requirements
+        based on the procedure being asserted.</p>
+    </section>
+
+    <section id="conformance-models">
+        <h2>Draft Conformance Approach</h2>
+        <p>WCAG 3.0 will use a different conformance model than WCAG 2.0 in order to meet its requirements. Developing
```
Comment (bruce-usab) [inline explainer/index.html:396]: This is the only appearance of WCAG 2.**_0_**.  I think it should be 2.2.

File: explainer/index.html [hunk 5]
```diff
@@ -0,0 +1,553 @@
...
+            and vetting the conformance model is a large portion of the work AG needs to complete over the next few
+            years. Drafts will include maturity models for public review and comment. </p>
+
+        <p>AG is exploring a model based on Foundational Requirements, Supplemental Requirements, and Assertions.</p>
+
+        <p>The most basic level of conformance will require meeting all of the Foundational Requirements. This set will be somewhat
+            comparable to WCAG 2.2 A & AA.</p>
```
Comment (bruce-usab) [inline explainer/index.html:403]: > This set will be somewhat comparable to WCAG 2.2 A & AA.

Suggest dropping "somewhat" qualifier.  I also suggest "WCAG 2.2 Level AA" or "all WCAG 2.2 Level A and Level AA success criteria".  I have been fighting the good fight that WCAG is more than the success criteria.

File: explainer/index.html [hunk 6]
```diff
@@ -0,0 +1,553 @@
...
+        <p>AG is currently <a href="https://www.w3.org/2023/11/ag-charter">chartered</a> through November 2025. The
+            schedule through that time period is maintained at <a
+                href="https://www.w3.org/WAI/GL/wiki/WCAG_3_Timeline#Publication_Plan">WCAG 3.0 Timeline</a>.</p>
+        <p>AG will continue to iterate within a single normative document with informative supporting documentation.
+            After exploring other options such as breaking the content's into modules, the group determined that the
+            fastest way forward at this time is to continue creating a single document with varying levels of maturity.
+            We will revisit whether mature content can be published as informative modules as part of the next charter
+            period, when content is more mature.</p>
+        <p>A final schedule will be delivered as part of this charter period. WCAG 3.0 will not be published until it
+            covers at least as much as WCAG 2.2.</p>
```
Comment (bruce-usab) [inline explainer/index.html:443]: Suggest "coverage comparable to WCAG 2.2 Level AA" and that the phrasing be consistent with the reference in 7. Draft Conformance Approach.

**Comment by lauracarlson** [comment]
I added a couple of editorial comments in the Google doc.

**Comment by detlevhfischer** [comment]
> Assertions may supplement Requirement to meet a Guideline. Not all Guidelines include Assertions. 

This sentence is odd and even seems slightly ungrammatical. Regarding its substance: Does that mean to conform you need to meet the testable requirement in any case, and are free to add an assertion? or does it mean you can meet a requirement by making an assertion? The text is not clear on that. If that lack of clarity is deliberate because the WG hasn't yet been decided whether or not a mere assertion can be deemed sufficient to meet a foundational requirement, this should be clearly stated.

It the meaning is that **some** requirements, due to the difficulty of testing them, can be met by an assertion but not others that **can** be tested, this should be clearly spelled out.

There is an empty bullet point in the Editor's note under 5.3.2 Documenting Assertions.

> The AG has discussed also including recommendations which could help improve accessibility under guidelines. 

This sentence feels wobbly. What does "accessibility under guidelines" mean? It's always about accessibility of content?

> will include additional or more mature examples
not quite clear what is meant.

File: .DS_Store
Comment (iadawn) [reply .DS_Store:1]: PR has been merged

File: explainer/index.html [hunk 7]
```diff
@@ -0,0 +1,553 @@
...
+                Assertion is an attributable and documented statement of fact regarding procedures practiced in the
+                development and maintenance of the content or product to improve accessibility.</p>
+    
+            <h4>Using Assertions</h4>
+            <p>Assertions may supplement Requirement to meet a Guideline. Not all Guidelines include Assertions. Those that do
+                list the Assertions with the Requirements. Organizations can make an assertion that they followed a procedure to
+                claim conformance. Results when testing assertions are true/false - the organization making the assertion
+                either provided the required documentation or it did not.</p>
+
+            <div class="ednote"></div>
```
[the remark this answers is not in the document]
Comment (iadawn) [reply explainer/index.html]: Fixed with 7e12833

File: explainer/index.html [hunk 8]
```diff
@@ -0,0 +1,553 @@
...
+                </ul>
+
+                <div class="ednote">
+                    <p>Within the current draft, Foundational Requirements are organized as a decision tree under the relevant Guidelines. 
+                        Supplemental Requirements and Assertions are then listed below the tree. AG hopes this will assist readers in 
+                        determining which Requirements and Assertions to use. The AG requests feedback on three
+                        early draft examples of this approach: Focus Appearance, Implied Meaning, and Text Alternatives.</p>
+                    <ul>
+                        <li>Does the organization make it easier to understand the Guideline, Requirements and Assertions?</li>
+                        <li></li>
```
[the remark this answers is not in the document]
Comment (iadawn) [reply explainer/index.html]: Fixed with 42c136f

File: explainer/index.html (hunk 5 above)
Comment (alastc) [reply explainer/index.html:401]: (Chair hat off)
I'd like a little wiggle-room that's created by the "somewhat". When requirements are re-configured there will be all sorts of cases where things aren't quite the same as before. We're aiming for similar coverage and effort-level, but a little wiggle room will be useful so people don't say "but it doesn't include this things from SC 1.3.1" or similar.

Adjusting to "WCAG 2.2 level AA" seems reasonable.

**Comment by alastc** [comment]
Hi @detlevhfischer,

> > Assertions may supplement Requirement to meet a Guideline. Not all Guidelines include Assertions. 
> This sentence is odd and even seems slightly ungrammatical. 

Would it be better as this?  "An Assertion is another approach to meeting a Guideline. They do not replace Requirements, and not all Guidelines include Assertions."

Then you said: 
> Regarding its substance: Does that mean to conform you need to meet the testable requirement in any case, and are free to add an assertion? Or does it mean you can meet a requirement (or just some requirements?) just by making an assertion? 

That's explained a little further down in section 7, where it says "Higher levels of conformance will be defined and met using Supplemental Requirements and Assertions."

So it's really down to the structure of the Foundational Requirements (baseline type things), then we've got the levels/percentages aspects above that.

> The text is not clear on that. If that lack of clarity is deliberate because the WG hasn't yet been decided whether or not a mere assertion can be deemed sufficient to meet a foundational requirement, this should be clearly stated.

I think if you look at the intro to assertions, and the intro to the conformance model, does that make sense to you?

> There is an empty bullet point in the Editor's note under 5.3.2 Documenting Assertions.

Thanks, I didn't see that in the preview, I'll check the code.

> > The AG has discussed also including recommendations which could help improve accessibility under guidelines. 
> This sentence feels wobbly. What does "accessibility under guidelines" mean? It's always about accessibility of content?

Fair point, and combining with my point above, how about we remove that from the Assertions ednote, and add an ednote to the 5.2 Requirements and methods section? 
E.g. "The AG has discussed also including 'Recommendations' at the same level as Supplemental Requirements. Recommendations would be things which could help improve accessibility but may not be objectively testable. They could contribute to higher levels of conformance."

> > will include additional or more mature examples
> not quite clear what is meant.

The full paragraph is: "AG will continue developing requirements and methods. Each publication will include additional or more mature examples for public comment."

That just means will will iterate on the guidelines and keep publishing. Either more examples, or the examples will increase in their level of maturity. I'm not seeing a better way of saying that off-hand.

File: explainer/index.html [hunk 9]
```diff
@@ -0,0 +1,551 @@
...
+
+                <div class="ednote">
+                    <p>Within the current draft, Foundational Requirements are organized as a decision tree under the relevant Guidelines. 
+                        Supplemental Requirements and Assertions are then listed below the tree. AG hopes this will assist readers in 
+                        determining which Requirements and Assertions to use. The AG requests feedback on three
+                        early draft examples of this approach: Focus Appearance, Implied Meaning, and Text Alternatives.</p>
+                    <ul>
+                        <li>Does the organization make it easier to understand the Guideline, Requirements and Assertions?</li>
+                        <li>The AG has discussed also including recommendations which could help improve accessibility under guidelines. 
+                            Recommendations are not be testable and would not contribute to conformance.</li>
```
Comment (alastc) [inline explainer/index.html:267]:
```suggestion
                            Recommendations are not testable and would not contribute to conformance.</li>
```

File: explainer/index.html [hunk 10]
```diff
@@ -0,0 +1,551 @@
...
+            </ul>   
+    </section>
+
+    <section id="schedule">
+        <h2>Schedule</h2>
+        <p>AG is currently <a href="https://www.w3.org/2023/11/ag-charter">chartered</a> through November 2025. The
+            schedule through that time period is maintained at <a
+                href="https://www.w3.org/WAI/GL/wiki/WCAG_3_Timeline#Publication_Plan">WCAG 3.0 Timeline</a>.</p>
+        <p>AG will continue to iterate within a single normative document with informative supporting documentation.
+            After exploring other options such as breaking the content's into modules, the group determined that the
```
Comment (alastc) [inline explainer/index.html:436]:
```suggestion
            After exploring other options such as breaking the content into modules, the group determined that the
```

**Comment by mraccess77** [comment]
> Assertion is another approach to meeting a Guideline. They do not replace Requirements

These 2 phrases seem contradictory to me.  I think what we are saying is that guidelines may necessitate either requirements and/or assertions be met.  That could be a combination or requirement(s) and assertion(s) or requirement(s) or assertion(s) depend on the guideline.  But what we aren't saying that instead of meeting a requirement you could make an assertion or vice versa (unless I misunderstand our approach).

**Comment by alastc** [comment]
Hi @mraccess77, we've made a few updates from the meeting, you can see them in [this commit](https://github.com/w3c/wcag3/pull/116/commits/a60c9d57fb3880c047d353a79f25be0ab4cfca97#diff-cbc10fb4d19472d5b1b3fdc6f39509c1f90afaba3822f0a4f18396175120a24cL217).

File: explainer/index.html (hunk 5 above)
Comment (iadawn) [reply explainer/index.html]: Discussed in [AG meeting on 2024-11-05](https://www.w3.org/2024/11/05-ag-minutes#41c5). Bruce accepted 'somewhat comparable'.

File: explainer/index.html (hunk 6 above)
Comment (iadawn) [reply explainer/index.html:453]: Not discussed in meeting of 2024-11-05. Resolving with no change but @bruce-usab please do raise an issue once I have merged this in if you still have concerns.

**Comment by wareid** [comment]
Very minor editorial/respec things, but it looks like a few words or terms are marked up as potentially having a glossary entry, but are not, so they get red squigglies underneath them:
* technology-agnostic
* programmatically
* significant change
* user activity

Even if we don't have definitions yet it may be worth having the terms in the glossary with "TBD" next to them to get rid of the errors. There is also an error with the link for "Documenting Assertions" that goes nowhere, I am assuming because we don't have that document to link to, so it might be worth removing the reference or specifying it's coming soon.

**Comment by alastc** [comment]
@wareid I'm not sure if it was in this PR, but Kevin added those definitions.

**Pull request merged**